pip_custom_platform/install.py

```diff
@@ -1,9 +1,11 @@
-# pylint:disable=import-error,no-member,no-name-in-module
 from __future__ import absolute_import
 from __future__ import unicode_literals
 
 import contextlib
-import distutils.util
+
+if True:
+    # pylint:disable=import-error
```

Comment:
this way we don't disable for the entire file. happy to revert this if you don't like it.